src/js/components/Pagination/stories/Steps.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Box, Pagination, Text } from 'grommet';
+
+export const Steps = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Box align="start" pad="small" gap="xlarge">
+    <Box fill="horizontal">
+      <Text>summary and stepSelector</Text>
+      <Pagination numberItems={237} stepOptions summary />
+    </Box>
+    <Box fill="horizontal">
+      <Text>stepOptions with custom steps</Text>
+      <Pagination numberItems={237} stepOptions={[10, 20, 30, 1000]} summary />
+    </Box>
+    <Box fill="horizontal">
+      <Text> stepOptions as boolean</Text>
+      <Pagination numberItems={237} stepOptions />
+    </Box>
```

Comment:
What is this story really attempting to demonstrate? Seems like we have the default stepOptions treatment accounted for in the first example.

If this is showing the stepOptions w/o summary, then we should consider reordering the way these examples unfold from simple to more complex.

Reply:
yes I was showing `stepOptions` w/o summary but since we don't have summary w/o `stepOptions` not sure this is needed but agree if keeping to reorder examples

Reply:
originally I was showing stepOptions w/o summary for us to test and see the responsive behavior so I kinda leaning to take it out @halocline @jcfilben thoughts?

Reply:
I don't have a strong preference either way. Removing is fine, but I can also see value in showing:
- Pagination with stepOptions
- Pagination with stepOptions and summary
- Pagination with custom step sizes

Reply:
done